addons/test_website/data/test_website_data.xml

```diff
@@ -95,5 +95,12 @@
             <div>I am a base view</div>
         </template>
 
+        <!-- RECORDS FOR REDIRECT TESTS -->
+        <template id="test_redirect_view">
+            <t t-esc="country.name"/>
+            <t t-if="not request.env.user._is_public()" t-esc="'Logged In'"/>
+            <a t-att-href="url_for('/test_website/country/andorra-1')">I am a link</a>
```

Comment:
can't we test classic rendering, with simple href instead ?

Url_for should be called automatically by _post_processing_att

Reply:
I didn't know/forgot `href` weren't send through `url_for` if you were logged in